nautobot/docs/user-guide/administration/configuration/optional-settings.md

```diff
@@ -487,6 +487,14 @@ Please see [the object permissions page](../guides/permissions.md) for more info
 
 ---
 
+## FILE_UPLOAD_MAX_MEMORY_SIZE
+
+Default:  2621440 (i.e. 2.5 MB).
+
+The maximum size (in bytes) that an upload will be before it gets streamed to the file system.
```

Comment:
I was thinking this should indicate the significance of this setting to Nautobot, rather than the generic Django documentation text.

Reply:
do you mean make it more Nautobot specific?

Reply:
Yes. Explain its relevance to Nautobot.